### ISSUE TITLE
feature: add `high contrast light themes` separator

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#12071]
---- Description

The proposal is to add the additional `high contrast light themes` or `light high contrast themes` separator to the themes quick-pick for accessibility reasons, and consistent. As a consequence the existing `high contrast themes` should be renamed to `high contrast dark themes` or `dark high contrast themes` respectfully.

The main argument is to more accurately group themes similarly to the `light themes` and `dark themes` groups, and better differentiate between user installed themes. At the moment all different types of high contrast themes are grouped under a common separator which can make their discovery more difficult as they will only be in alphabetical order.

https://github-com/microsoft/vscode/blob/2fe850b690fd4e9f53c86055fa7aedd825800ea6/src/vs/workbench/contrib/themes/browser/themes.contribution.ts-L386-L390

_Before_:

![image](https://user-images.githubusercontent.com/40359487/202753400-0588be67-d133-4171-922a-ad37f89602a2.png)

_After_:

![image](https://user-images.githubusercontent.com/40359487/202757936-63dca625-d91c-4212-b076-9ced597522d0.png)
